resources/compass-runtime-agent/templates/deployment.yaml

```diff
@@ -19,15 +19,36 @@ spec:
     metadata:
       annotations:
         sidecar.istio.io/inject: "true"
+        container.apparmor.security.beta.kubernetes.io: runtime/default
       labels:
         app: {{ .Chart.Name }}
         release: {{ .Release.Name }}
     spec:
+      securityContext:
+        runAsUser: 65535
+        runAsGroup: 65535
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
       serviceAccountName: {{ .Chart.Name }}
       containers:
         - name: {{ .Chart.Name }}
+          securityContext:
+            privileged: false
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
+            capabilities:
+              drop:
+                - ALL
+              add:
+                - NET_BIND_SERVICE
```

Comment:
Is that capability really needed? If not the "ad" section should be removed

Reply:
@franpog859 The capability `NET_BIND_SERVICE` is the only one which is allowed for restricted Pod Security Standard. Adding this is optional and only necessary if you really need this capability. Otherwise please remove the whole `add` section.